drizzle-kit/src/serializer/sqliteSerializer.ts

```diff
@@ -696,10 +783,48 @@ WHERE
 		progressCallback('enums', 0, 'done');
 	}
 
+	const views = await db.query(
+		`SELECT name AS view_name, sql AS sql FROM sqlite_master WHERE type = 'view';`,
+	);
+
+	viewsCount = views.length;
+
+	if (progressCallback) {
+		progressCallback('views', viewsCount, 'fetching');
+	}
+	for (const view of views) {
+		const viewName = view['view_name'];
+		const sql = view['sql'];
+
+		const regex = new RegExp(`\\bAS\\b\\s+(SELECT.+)$`, 'i');
+		const match = sql.match(regex);
+
+		if (!match) {
+			console.log('Could not process view');
+			process.exit(1);
+		}
```

Comment:
This regex breaks if I have any views in my database that are defined on more than one line or somewhat complex e.g. with a CTE:

```sql
CREATE VIEW latest_user_logs AS WITH ranked_logs AS ( SELECT
        ul.id,
        ul.user_id,
        ul.entry,
        ul.dt_timestamp,
        date(datetime(dt_timestamp, 'unixepoch')) AS log_date,
        ROW_NUMBER() OVER (
            PARTITION BY ul.user_id, ul.type, date(datetime(dt_timestamp, 'unixepoch'))
            ORDER BY ul.dt_created DESC
        ) AS rn
    FROM user_logs ul
)
SELECT id, user_id, entry, log_date, dt_timestamp
FROM ranked_logs
WHERE rn = 1
ORDER BY log_date DESC;
```

I got a very dubious error even in verbose mode that was hard to track down:

```
> drizzle-kit push --verbose

No config path provided, using default 'drizzle.config.ts'
[⣷] Pulling schema from database...
Could not process view
```

At the very least here we could put a better log message, that includes the view name and sql:

```javascript
		if (!match) {
			console.log(`Could not process view, ${viewName}`);
			console.debug(`${viewName} SQL`, sql);
			process.exit(1);
		}
```

But it would be good to fix the regex entirely.

Workaround at the moment is I changed my view to a subquery and put it all on one line.